Plugins/Public/playercntl_plugin/Main.cpp

```diff
@@ -428,6 +437,18 @@ namespace HkIServerImpl
 		}
 	}
 
+	void __stdcall RequestCancel(int iType, unsigned int iShip, unsigned int p3, unsigned long p4, unsigned int iClientID)
+	{
+		returncode = DEFAULT_RETURNCODE;
+
+		if (!iClientID)
+		{
+			return;
+		}
+
+		HyperJump::RequestCancel(iType, iShip, p3, p4);
```

Comment:
we not also going to send iClientID here?

also, p3 has a name in the called function, so I expect it to have a name here too

Reply:
Problem with RequestEvent and RequestCancel is that the parameters have different meanings depending on the value of the first parameter, hence why I haven't named the variables in the main.cpp, but did in the HyperJump subcall, where we only care about a very specific event type == 0 aka dock request